src/components/Map/mapStyle.ts

```diff
@@ -1,9 +1,7 @@
-import { StyleSpecification } from "maplibre-gl";
-
 export default function mapStyle() {
```

Comment:
Can’t you keep the type here like this? 🤔 

```suggestion
import { StyleSpecification } from "maplibre-gl";

export default function mapStyle(): StyleSpecification {
```

… and I would also call this `getMapStyle()` or so, to make it clear that this is a function and _does_ something.

Reply:
Makes sense!